### PR TITLE
Add compat data for MutationObserver

### PR DESCRIPTION
**Pull request by bunnybooboo**
https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver

### CODE REVIEW

File: api/MutationObserver.json [hunk 1]
```diff
@@ -0,0 +1,473 @@
+{
+  "api": {
+    "MutationObserver": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/MutationObserver",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"webview_android": [
  {
    "version_added": "26"
  },
  {
    "version_added": "18",
    "version_removed": "26",
    "prefix": "Webkit"
  }
]
```
And moved to the end of the `support` block (see #398 and #1882).

File: api/MutationObserver.json [hunk 2]
```diff
@@ -0,0 +1,473 @@
...
+            },
+            {
+              "version_added": "7"
+            }
+          ],
+          "safari_ios": [
+            {
+              "version_added": "6",
+              "version_removed": "7",
+              "notes": "Prefixed with <code>-webkit</code>"
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"safari_ios": [
  {
    "version_added": "7"
  },
  {
    "version_added": "6",
    "version_removed": "7",
    "prefix": "Webkit"
  }
]
```

File: api/MutationObserver.json [hunk 3]
```diff
@@ -0,0 +1,473 @@
...
+            },
+            {
+              "version_added": "7"
+            }
+          ],
+          "samsunginternet_android": {
+            "version_added": null
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: `true`

File: api/MutationObserver.json [hunk 4]
```diff
@@ -0,0 +1,473 @@
...
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": true
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: `false`

File: api/MutationObserver.json [hunk 5]
```diff
@@ -0,0 +1,473 @@
...
+        }
+      },
+      "MutationObserver": {
+        "__compat": {
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"webview_android": [
  {
    "version_added": "26"
  },
  {
    "version_added": "18",
    "version_removed": "26",
    "prefix": "Webkit"
  }
]
```
And moved to the end of the `support` block (see #398 and #1882).

File: api/MutationObserver.json [hunk 6]
```diff
@@ -0,0 +1,473 @@
...
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": true
+        }
+      },
+      "MutationObserver": {
+        "__compat": {
+          "support": {
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: Add
```json
"description": "<code>MutationObserver()</code> constructor",
"mdn_url": "https://developer.mozilla.org/docs/Web/API/MutationObserver/MutationObserver",
```
before this line.

File: api/MutationObserver.json [hunk 7]
```diff
@@ -0,0 +1,473 @@
...
+            "version_added": "15"
+          },
+          "opera_android": {
+            "version_added": "15"
+          },
+          "safari": [
+            {
+              "version_added": "6",
+              "version_removed": "7",
+              "notes": "Prefixed with <code>-webkit</code>"
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"safari": [
  {
    "version_added": "7"
  },
  {
    "version_added": "6",
    "version_removed": "7",
    "prefix": "Webkit"
  }
]
```

File: api/MutationObserver.json [hunk 8]
```diff
@@ -0,0 +1,473 @@
...
+            },
+            {
+              "version_added": "26"
+            }
+          ],
+          "chrome_android": [
+            {
+              "version_added": "18",
+              "version_removed": "26",
+              "notes": "Prefixed with <code>-webkit</code>."
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"chrome_android": [
  {
    "version_added": "26"
  },
  {
    "version_added": "18",
    "version_removed": "26",
    "prefix": "Webkit"
  }
]
```

File: api/MutationObserver.json [hunk 9]
```diff
@@ -0,0 +1,473 @@
...
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/MutationObserver",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": [
+            {
+              "version_added": "18",
+              "version_removed": "26",
+              "notes": "Prefixed with <code>-webkit</code>."
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"chrome": [
  {
    "version_added": "26"
  },
  {
    "version_added": "18",
    "version_removed": "26",
    "prefix": "Webkit"
  }
]
```

File: api/MutationObserver.json [hunk 10]
```diff
@@ -0,0 +1,473 @@
...
+        "__compat": {
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": [
+              {
+                "version_added": "18",
+                "version_removed": "26",
+                "notes": "Prefixed with <code>-webkit</code>."
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"chrome": [
  {
    "version_added": "26"
  },
  {
    "version_added": "18",
    "version_removed": "26",
    "prefix": "Webkit"
  }
]
```

File: api/MutationObserver.json [hunk 11]
```diff
@@ -0,0 +1,473 @@
...
+              },
+              {
+                "version_added": "26"
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "18",
+                "version_removed": "26",
+                "notes": "Prefixed with <code>-webkit</code>."
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"chrome_android": [
  {
    "version_added": "26"
  },
  {
    "version_added": "18",
    "version_removed": "26",
    "prefix": "Webkit"
  }
]
```

File: api/MutationObserver.json [hunk 12]
```diff
@@ -0,0 +1,473 @@
...
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": "15"
+            },
+            "safari": [
+              {
+                "version_added": "6",
+                "version_removed": "7",
+                "notes": "Prefixed with <code>-webkit</code>"
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"safari": [
  {
    "version_added": "7"
  },
  {
    "version_added": "6",
    "version_removed": "7",
    "prefix": "Webkit"
  }
]
```

File: api/MutationObserver.json [hunk 13]
```diff
@@ -0,0 +1,473 @@
...
+              },
+              {
+                "version_added": "7"
+              }
+            ],
+            "safari_ios": [
+              {
+                "version_added": "6",
+                "version_removed": "7",
+                "notes": "Prefixed with <code>-webkit</code>"
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"safari_ios": [
  {
    "version_added": "7"
  },
  {
    "version_added": "6",
    "version_removed": "7",
    "prefix": "Webkit"
  }
]
```

File: api/MutationObserver.json [hunk 14]
```diff
@@ -0,0 +1,473 @@
...
+              },
+              {
+                "version_added": "7"
+              }
+            ],
+            "samsunginternet_android": {
+              "version_added": null
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: `true`

File: api/MutationObserver.json [hunk 15]
```diff
@@ -0,0 +1,473 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": true
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: `false`

File: api/MutationObserver.json [hunk 16]
```diff
@@ -0,0 +1,473 @@
...
+          }
+        }
+      },
+      "observe": {
+        "__compat": {
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"webview_android": {
  "version_added": "18"
}
```
And moved to the end of the `support` block (see #398 and #1882).

File: api/MutationObserver.json [hunk 17]
```diff
@@ -0,0 +1,473 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": true
+          }
+        }
+      },
+      "observe": {
+        "__compat": {
+          "support": {
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: Add `"mdn_url": "https://developer.mozilla.org/docs/Web/API/MutationObserver/observe"` before this line.

File: api/MutationObserver.json [hunk 18]
```diff
@@ -0,0 +1,473 @@
...
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": [
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"chrome": {
  "version_added": "18"
}
```

File: api/MutationObserver.json [hunk 19]
```diff
@@ -0,0 +1,473 @@
...
+              {
+                "version_added": "18",
+                "version_removed": "26",
+                "notes": "Prefixed with <code>-webkit</code>."
+              },
+              {
+                "version_added": "26"
+              }
+            ],
+            "chrome_android": [
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"chrome_android": {
  "version_added": "18"
}
```

File: api/MutationObserver.json [hunk 20]
```diff
@@ -0,0 +1,473 @@
...
+            "ie": {
+              "version_added": "11"
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": "15"
+            },
+            "safari": [
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"safari": {
  "version_added": "6"
}
```

File: api/MutationObserver.json [hunk 21]
```diff
@@ -0,0 +1,473 @@
...
+              {
+                "version_added": "6",
+                "version_removed": "7",
+                "notes": "Prefixed with <code>-webkit</code>"
+              },
+              {
+                "version_added": "7"
+              }
+            ],
+            "safari_ios": [
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"safari_ios": {
  "version_added": "6"
}
```

File: api/MutationObserver.json [hunk 22]
```diff
@@ -0,0 +1,473 @@
...
+              }
+            ],
+            "samsunginternet_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": true
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: `false`

File: api/MutationObserver.json [hunk 23]
```diff
@@ -0,0 +1,473 @@
...
+          }
+        }
+      },
+      "disconnect": {
+        "__compat": {
+          "support": {
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: Add `"mdn_url": "https://developer.mozilla.org/docs/Web/API/MutationObserver/disconnect"` before this line.

File: api/MutationObserver.json [hunk 24]
```diff
@@ -0,0 +1,473 @@
...
+            "webview_android": {
+              "version_added": null
+            },
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"webview_android": {
  "version_added": "18"
}
```
And moved to the end of the `support` block (see #398 and #1882).

File: api/MutationObserver.json [hunk 25]
```diff
@@ -0,0 +1,473 @@
...
+            "chrome": [
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"chrome": {
  "version_added": "18"
}
```

File: api/MutationObserver.json [hunk 26]
```diff
@@ -0,0 +1,473 @@
...
+              {
+                "version_added": "18",
+                "version_removed": "26",
+                "notes": "Prefixed with <code>-webkit</code>."
+              },
+              {
+                "version_added": "26"
+              }
+            ],
+            "chrome_android": [
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"chrome_android": {
  "version_added": "18"
}
```

File: api/MutationObserver.json [hunk 27]
```diff
@@ -0,0 +1,473 @@
...
+            "ie": {
+              "version_added": "11"
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": "15"
+            },
+            "safari": [
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"safari": {
  "version_added": "7"
}
```

File: api/MutationObserver.json [hunk 28]
```diff
@@ -0,0 +1,473 @@
...
+              {
+                "version_added": "6",
+                "version_removed": "7",
+                "notes": "Prefixed with <code>-webkit</code>"
+              },
+              {
+                "version_added": "7"
+              }
+            ],
+            "safari_ios": [
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"safari_ios": {
  "version_added": "7"
}
```

File: api/MutationObserver.json [hunk 29]
```diff
@@ -0,0 +1,473 @@
...
+                "version_added": "6",
+                "version_removed": "7",
+                "notes": "Prefixed with <code>-webkit</code>"
+              },
+              {
+                "version_added": "7"
+              }
+            ],
+            "samsunginternet_android": {
+              "version_added": null
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: `true`

File: api/MutationObserver.json [hunk 30]
```diff
@@ -0,0 +1,473 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": true
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: `false`

File: api/MutationObserver.json [hunk 31]
```diff
@@ -0,0 +1,473 @@
...
+          }
+        }
+      },
+      "takeRecords": {
+        "__compat": {
+          "support": {
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: Add `"mdn_url": "https://developer.mozilla.org/docs/Web/API/MutationObserver/takeRecords"` before this line.

File: api/MutationObserver.json [hunk 32]
```diff
@@ -0,0 +1,473 @@
...
+            "webview_android": {
+              "version_added": null
+            },
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"webview_android": {
  "version_added": "18"
}
```
And moved to the end of the `support` block (see #398 and #1882).

File: api/MutationObserver.json [hunk 33]
```diff
@@ -0,0 +1,473 @@
...
+            "chrome": [
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"chrome": {
  "version_added": "18"
}
```

File: api/MutationObserver.json [hunk 34]
```diff
@@ -0,0 +1,473 @@
...
+              {
+                "version_added": "18",
+                "version_removed": "26",
+                "notes": "Prefixed with <code>-webkit</code>."
+              },
+              {
+                "version_added": "26"
+              }
+            ],
+            "chrome_android": [
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"chrome_android": {
  "version_added": "18"
}
```

File: api/MutationObserver.json [hunk 35]
```diff
@@ -0,0 +1,473 @@
...
+            "ie": {
+              "version_added": "11"
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": "15"
+            },
+            "safari": [
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"safari": {
  "version_added": "7"
}
```

File: api/MutationObserver.json [hunk 36]
```diff
@@ -0,0 +1,473 @@
...
+              {
+                "version_added": "6",
+                "version_removed": "7",
+                "notes": "Prefixed with <code>-webkit</code>"
+              },
+              {
+                "version_added": "7"
+              }
+            ],
+            "safari_ios": [
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This should be:
```json
"safari_ios": {
  "version_added": "7"
}
```

File: api/MutationObserver.json [hunk 37]
```diff
@@ -0,0 +1,473 @@
...
+                "version_added": "6",
+                "version_removed": "7",
+                "notes": "Prefixed with <code>-webkit</code>"
+              },
+              {
+                "version_added": "7"
+              }
+            ],
+            "samsunginternet_android": {
+              "version_added": null
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: `true`

File: api/MutationObserver.json [hunk 38]
```diff
@@ -0,0 +1,473 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": true
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: `false`

File: api/MutationObserver.json [hunk 39]
```diff
@@ -0,0 +1,473 @@
...
+          }
+        }
+      },
+      "MutationObserverInit": {
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This has been done by #2299 in a much better way, so remove this.

**Comment by ExE-Boss** [comment]
@a2sheppy

**Comment by Elchi3** [comment]
I've updated this, thank you for the review @ExE-Boss! Let me know if this looks good now and I'll merge it before I release 0.0.40 later this evening.

File: api/MutationObserver.json [hunk 40]
```diff
@@ -0,0 +1,330 @@
...
+              "version_added": "11"
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": "15"
+            },
+            "safari": {
+              "version_added": "7"
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This was `6`.

File: api/MutationObserver.json [hunk 41]
```diff
@@ -0,0 +1,330 @@
...
+            },
+            "safari_ios": {
+              "version_added": "7"
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This was `6`.

File: api/MutationObserver.json [hunk 42]
```diff
@@ -0,0 +1,330 @@
...
+              "version_added": "11"
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": "15"
+            },
+            "safari": {
+              "version_added": "7"
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This was `6`.

File: api/MutationObserver.json [hunk 43]
```diff
@@ -0,0 +1,330 @@
...
+            },
+            "safari_ios": {
+              "version_added": "7"
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: This was `6`.

File: api/MutationObserver.json [hunk 44]
```diff
@@ -0,0 +1,330 @@
...
+              "version_added": "14"
+            },
+            "ie": {
+              "version_added": "11"
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": "15"
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: Opera Android began using WebKit/Blink with version `14`, 1 version before Opera for Desktop.

File: api/MutationObserver.json [hunk 45]
```diff
@@ -0,0 +1,330 @@
...
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: ditto

File: api/MutationObserver.json [hunk 46]
```diff
@@ -0,0 +1,330 @@
...
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: ditto

File: api/MutationObserver.json [hunk 47]
```diff
@@ -0,0 +1,330 @@
...
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: Opera Android began using WebKit/Blink with version `14`, 1 version before Opera for Desktop.

File: api/MutationObserver.json [hunk 48]
```diff
@@ -0,0 +1,330 @@
...
+            "version_added": "14"
+          },
+          "ie": {
+            "version_added": "11"
+          },
+          "opera": {
+            "version_added": "15"
+          },
+          "opera_android": {
+            "version_added": "15"
```
Comment (ExE-Boss) [inline api/MutationObserver.json]: Opera Android began using WebKit/Blink with version `14`, 1 version before Opera for Desktop.

**Comment by Elchi3** [comment]
Well spotted, thanks again!

**Pull request merged**